editor/components/inserter/item-list.js

```diff
@@ -74,7 +74,15 @@ class ItemList extends Component {
 						<button
 							role="menuitem"
 							key={ item.id }
-							className={ classnames( 'editor-inserter__item', getBlockMenuDefaultClassName( item.id ) ) }
+							className={
+								classnames(
+									'editor-inserter__item',
+									getBlockMenuDefaultClassName( item.id ),
+									{
+										'editor-inserter__item-with-childs': item.hasChildBlocks,
```

Comment:
- Grammar: "childs" -> "children"
- This is a modifier, so could be written as one, e.g. `has-children` ([reference](https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#naming))